src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -934,7 +934,11 @@ public Visitable visit(MethodDeclaration node, Void arg) {
     } else if (insideTargetMember) {
       return processMethodDeclaration(node);
     } else {
-      return super.visit(node, arg);
+      // Do not call super.visit(): this method is definitely unused by the targets, and so
```

Comment:
Wow, this is quite a subtle bug. Thanks for pointing it out.

Reply:
I'm not sure that this fix is really the correct one. Would it be better to move all of the data structure maintenance from `processMethodDeclaration` to this visit method, instead?

I'm a bit concerned that not calling `super.visit()` here will lead to some other subtle bug in the future, but I can't articulate exactly why it makes me nervous.

Reply:
It seems fine to me, since we are not using any local variable in those methods, and the data structure maintenance in `processMethodDeclaration` is simply for updating the list of local variables.